Zend/tests/request_parse_body/multipart_options_invalid_key.phpt

```diff
@@ -0,0 +1,21 @@
+--TEST--
+request_parse_body() invalid key
+--FILE--
+<?php
+
+try {
+    request_parse_body(options: ['foo' => 1]);
+} catch (Error $e) {
```

Comment:
Nit: It would be nice to assert the exact exceptions which are thrown. E.g. `ValueError`, `RequestParseBodyException `, `InvalidArgumentException` etc.